ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -55,6 +63,27 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ipi-aws
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        MAKEFILE_TARGET="create_full_environment run"
```

Comment:
Why did you override the default Makefile target? we would like to verify a full installation.

Reply:
I will test this workflow but I have been testing a simplified flow for these jobs that doesn't time out.